modules/policy-criteria.adoc

```diff
@@ -13,7 +13,9 @@ You can configure the policy based on the attributes listed in the following tab
 In this table:
 
 * The *Regular expressions*, *AND, OR*, and *NOT* columns indicate whether you can use regular expressions and other logical operators along with the specific attribute.
-`!` in the *Regular expressions* column indicates that you can only use regular expressions for the listed fields.
+** `!` in the *Regular expressions* column indicates that you can only use regular expressions for the listed fields.
+** `!` in the *AND, OR* column indicates that you can only use the mentioned logical operator for that attribute.
+* The *RHACS version* column indicates the version of {product-title} you must have to use that attribute.
```

Comment:
To me, this line reads a little awkwardly.  Maybe "...of {product-title} that you must have to use the attribute."  If you agree, consider s/that attribute/the attribute/ on line 17 for parallelism. Up to you.

Unsolicited feedback on unmodified content:
* In other places on the page, AND and OR are identified as logical operators and not logical combination operators.  Sharing in case that subtle difference isn't intentional.
* Also, ISG discourages parens in general text. I don't have a page number handy, but can get one if you like. Elsewhere on the page, the operators are simply separated with commas rather than parens.

Reply:
Thank you @mikemckiernan for reviewing this. I've changed the following:
- Updated the lines as suggested
- Removed all parens 🙇🏼